app/controllers/home_controller.rb

```diff
@@ -48,6 +49,10 @@ def load_waiting_applications
     end
   end
 
+  def load_users_last_applications
+    @last_updated_applications = Query::LastUpdatedApplications.new(current_user).find(limit: 5)
```

Comment:
I forgot to mention we may also need to memoize the `@last_updated_applications`.

Reply:
sure